src/closestPair.js

```diff
@@ -0,0 +1,26 @@
+export default function closestPair(points) {
+
+  let lowestDistance = pythagorean(points[0], points[1])
+  let closestPoints = []
+
+  for (let i = 0; i < points.length; i++) {
+    for (let j = 0; j < points.length; j++) {
+      if(points[i] !== points[j] && pythagorean(points[i], points[j]) < lowestDistance) {
```

Comment:
you should store the result of `pythagorean(points[i], points[j])` in a variable rather than running it again on the next line down